packages/boot-loader/lib/index.ts

```diff
@@ -76,7 +76,7 @@ export class BootLoader {
         this.logger.trace("boot.afterBootFunctions.start", fnObj.name);
       }
     } catch (err) {
-      process.stderr.write(`BootLoader.boot.error.caught: ${err}\n`);
+      this.logger.error(`BootLoader.boot.error.caught: ${err}\n`);
```

Comment:
Does it work this way? I believe, I wanted to keep the logger out of the boot-loader to reduce it's dependencies, being the root package.

Reply:
It does work. We only have the following dependencies among boot-loader, logger and config:
boot-loader -> logger
logger -> config
It does make sense for the boot-loaders output.